internal/sorted/doc.go

```diff
@@ -0,0 +1,6 @@
+// Copyright ©2024 The Gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Package sorted provides common sorting functions.
+package sorted // import "gonum.org/v1/gonum/internal/ordered"
```

Comment:
Why is the package name changed?

Reply:
> Why is the package name changed?

@kortschak To stop it conflicting with `gonum.org/v1/gonum/graph/internal/ordered` when both packages are imported in the same file. The alternative is to use an import alias. Any preferences? Any other ideas, by any chance?

But if we're happy to proceed, I'll make sure to fix that `// import "..."` comment, as it's documenting a non-existent package now. 😅

Reply:
Resolving because we're moving everything in `graph/internal/ordered` to `internal/order` package.